docs/quickstarts/ratify-on-azure.md

```diff
@@ -166,6 +168,45 @@ Configure user-assigned managed identity and enable `AcrPull` role to the worklo
     kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml
     kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
     ```
+### When Azure Policy Addon is enabled on AKS
+1. Ensure your AKS cluster is 1.26+
+2. `az feature register -n AKS-AzurePolicyExternalData --namespace Microsoft.ContainerService`. If the feature flag is first to register, you need to disable and re-enable azure policy addon.
```

Comment:
suggestion for update. " If the feature flag was registered before the addon , you need to disable and re-enable azure policy addon."

Reply:
remove this word since it is fixed for now